src/Core/Scaffold/PayloadBuilder.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace GetCandy\Api\Core\Scaffold;
+
+use Illuminate\Support\Str;
+use Illuminate\Database\Eloquent\Model;
+use Schema;
+use Illuminate\Validation\Validator;
+use Illuminate\Contracts\Validation\Factory;
+
+class PayloadBuilder
+{
+    protected $payload;
+
+    public function __construct(PayloadInterface $payload = null)
+    {
+        $this->setPayload($payload);
+    }
+
+    public function validate(array $data, array $messages = [], array $customAttributes = [])
+    {
+        return $this->getValidationFactory()->make(
+            $data, $this->payload->rules, $messages, $customAttributes
+        )->validate();
+    }
+
+    public function prepare($request)
```

Comment:
Change this to `load()` and allow both HTTP Requests and Arrays to be passed through.

e.g. 

```
$payload = NewAddressPayload::load($request);
$payload = NewAddressPayload::load($dataArray);
```

https://laravel.com/docs/7.x/helpers#method-array-only
```
$input = $request->only(['username', 'password']);
```